src/iiif/views.py

```diff
@@ -22,6 +22,7 @@ def index(request, iiif_url):
         file_response, file_url = tools.get_file(request.META, url_info, iiif_url, metadata)
         tools.handle_file_response_errors(file_response, file_url)
     except tools.ImmediateHttpResponse as e:
+        log.error(str(e.response.content))
```

Comment:
`log.exception("Some useful description")` is hier speciaal voor gemaakt. Die pakt de stacktrace info zelf.